Node-JS-Projects/Url-Scraper/README.md

```diff
@@ -0,0 +1,97 @@
+<h1 align='center'><b>💥 PROJECT NAME 💥</b></h1>
```

Comment:
![image](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/b61a257d-859a-4818-be55-8f6b72757d93)

Update the project name `URL-SCRAPER` in readme file.